index.js

```diff
@@ -1,80 +1,97 @@
 
-const books = getBooks();
+    const bookDisplay = document.createElement('div');
+    bookDisplay.className = 'Collection1';
+    bookDisplay.innerHTML = `
+      <p class="bookTitle">${book.title} by ${book.author}</p>
+      <button class="delete">Remove</button>
+    `;
 
-function displayBooks() {
-  books.forEach((book) => addBookToList(book));
-}
+    list.appendChild(bookDisplay);
+  }
 
-document.addEventListener('DOMContentLoaded', displayBooks);
+  /* eslint-disable-next-line */
+  getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
 
-function addBook(book) {
-  const books = getBooks();
-  books.push(book);
-  localStorage.setItem('books', JSON.stringify(books));
-}
+    return books;
+  }
 
-function removeBook(title) {
-  const books = getBooks();
+  displayBooks() {
+    this.books.forEach((book) => this.addBookToList(book));
+  }
 
-  books.forEach((book, index) => {
-    if (book.title === title) {
-      books.splice(index, 1);
-    }
-  });
+  addBook(book) {
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
 
-  localStorage.setItem('books', JSON.stringify(books));
-}
+  removeBook(title) {
+    this.books = this.books.filter((book) => book.title !== title);
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
 
-function clearFields() {
-  document.querySelector('#title').value = '';
-  document.querySelector('#author').value = '';
-}
+  /* eslint-disable-next-line */
+  clearFields() {
+    document.querySelector('#title').value = '';
+    document.querySelector('#author').value = '';
+  }
 
-document.querySelector('#book-form').addEventListener('submit', (event) => {
-  event.preventDefault();
+  handleFormSubmit(event) {
+    event.preventDefault();
 
-  const title = document.querySelector('#title').value;
-  const author = document.querySelector('#author').value;
+    const title = document.querySelector('#title').value;
+    const author = document.querySelector('#author').value;
 
-  const book = {
-    title,
-    author,
-  };
+    const book = new Book(title, author);
 
-  addBookToList(book);
-  addBook(book);
-  clearFields();
-});
+    this.addBookToList(book);
+    this.addBook(book);
+    this.clearFields();
+  }
 
-document.querySelector('#collection').addEventListener('click', (event) => {
-  if (event.target.classList.contains('delete')) {
-    event.target.parentElement.remove();
-    const bookTitle = event.target.previousElementSibling.previousElementSibling.textContent;
-    removeBook(bookTitle);
+  handleCollectionClick(event) {
+    if (event.target.classList.contains('delete')) {
+      event.target.parentElement.remove();
+      const bookTitle = event.target.previousElementSibling.previousElementSibling.textContent;
+      this.removeBook(bookTitle);
+    }
```

Comment:
- [ ] Kindly investigate your remove books functionality. It only removes the book from the page, but not from the local storage.